modules/roxctl-declarative-config-create-notifier-splunk.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * command-reference/roxctl-declarative-config.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="roxctl-declarative-config-create-notifier-splunk_{context}"]
+= roxctl declarative-config create notifier splunk
+
+Create a declarative configuration for a splunk notifier.
+
+.Usage
+[source,terminal]
+----
+$ roxctl declarative-config create notifier splunk [flags]
+----
+
+.Options
+[cols="2,2",options="header"]
+|===
+|Option |Description
+
+|`--audit-logging`
+|Enable audit logging. The default value is `false`.
+
+|`--source-types stringToString`
+|Specify Splunk source types as comma-separated `key=value` pairs. The default value is `[]`.
+
+|`--splunk-endpoint string`
+|Specify the Splunk HTTP endpoint. This is a mandatory option.
+
+|`--splunk-skip-tls-verify`
+|Insecure connection to Splunk. The default value is `false`.
```

Comment:
```suggestion
|Defines an insecure connection to Splunk. The default value is `false`.
```

Reply:
Jw does this make sense in context? Trying to keep this verb tense.

Reply:
@dhaus67 ??

Reply:
Hmm, I suppose the more fitting would be `Use an insecure connection` instead of `Defines`. This really just tells people that they don't **use** HTTPS but rather HTTP, so it seems like the better fit over `Defines`.